docs/reference/commandline/run.md

```diff
@@ -143,7 +143,7 @@ previous changes intact using `docker start`. See `docker ps -a` to view a list
 of all containers.
 
 The `docker run` command can be used in combination with `docker commit` to
-[*change the command that a container runs*](commit.md). There is additional detailed information about `docker run` in the [Docker run reference](../run.md).
```

Comment:
This is linking to a _different_ `run.md` document https://github.com/docker/docker/blob/1.12.x/docs/reference/run.md, so that link should not be removed